go/test/endtoend/onlineddl/ghost/onlineddl_ghost_test.go

```diff
@@ -221,11 +223,11 @@ func TestSchemaChange(t *testing.T) {
 
 	testWithInitialSchema(t)
 	t.Run("create non_online", func(t *testing.T) {
-		_ = testOnlineDDLStatement(t, alterTableNormalStatement, string(schema.DDLStrategyDirect), "vtctl", "non_online", "")
+		_ = testOnlineDDLStatement(t, alterTableNormalStatement, "direct", "vtctl", "non_online", "")
```

Comment:
Can we have a helper function that translates `OnlineDDL_DIRECT` to the string `"direct"`?

Reply:
Presumably based on `tabletmanagerdatapb.OnlineDDL_Strategy_name[int32(strategy)]`?